docs/faq.rst

```diff
@@ -24,6 +26,36 @@ comparing `jinja2`_ to `Django`_.
 .. _jinja2: http://jinja.pocoo.org/
 .. _Django: https://www.djangoproject.com/
 
+How can I use Scrapy with BeautifulSoup?
+----------------------------------------
+
+As mentioned :ref:`above <faq-scrapy-bs-cmp>`, BeautifulSoup can be used
+for parsing HTML responses in Scrapy callbacks.
+You just have to feed the response's body into a ``BeautifulSoup`` object
+and extract whatever data you need from it.
+
+Here's an example spider using ``lxml`` parser with BeautifulSoup API::
```

Comment:
I think it is better to keep the example minimal; it may worths adding a note about 'lxml' though - if I'm not mistaken, it is not default because lxml is hard to install, but if Scrapy is installed then lxml is already installed.